### ISSUE TITLE
Uniformize local number field -> p-adic field

### ISSUE THREAD

**Issue by AndrewVSutherland** [LMFDB/lmfdb#4516]
There are still a few places where we still use "local number field" rather than "p-adic field" (e.g. "local field labels" heading in the learn more about box).

We should also create a new route (e.g. pAdicField and redirect LocalNumberField).

**Comment by AndrewVSutherland** [comment]
This has been implemented on beta (via PR #4510) and I was about to close it, but I notice that the old links don't get redirected.  

I think it might be better to redirect them so that the URLs are consistent -- it might be confusing to people otherwise, who might wonder whether there is any difference between the pages

https://beta.lmfdb.org/LocalNumberField/149.5.0.1
https://beta.lmfdb.org/pAdicField/149.5.0.1

What do you think @jwj61 ?  Is there a reason you didn't want to do a redirect?

**Comment by jwj61** [comment]
I thought adding the new url prefix padicField was an improvement, but that at some point we should add redirects.  Note, I used a lower case a since I thought that one writes $p$-adic  and rarely, if ever, $p$-Adic.

The pull request which added padicField url was #4518.  It does say there that it only added the second url prefix, but did not do redirects, but only because that would be more involved.  I was thinking I would need to create a redirect for every route, but maybe it can be done with just one.  I am not sure what the most efficient way to do it would be.

**Comment by AndrewVSutherland** [comment]
@jwj61 Thanks for the corrections/clarifications, sorry I read the wrong PR, and I think I agree on padic rather than pAdic, I'm not sure why my brain wants to camel case everything (but if anyone thinks it should be pAdic, speak up now or forever hold your peace).  @roed314 and @edgarcosta is there any easy way to handle the redirects in one place rather than on every route?

Also, @edgarcosta we need to add padicField to the list of production routes so it doesn't redirect to beta.

**Comment by edgarcosta** [comment]
To avoid the `www -> beta` redirect you need to add `padicField` to https://github.com/LMFDB/lmfdb/blob/master/lmfdb/app.py#L665

For the redirect, LocalNumberField -> padicField add a function in `__init__.py` along the lines:
```
@local_fields_page.before_request
def redirect_local():
        urlparts = urlparse(request.url)
        urlparts = urlparts._replace(path=urlparts.replace('LocalNumberField', 'padicField'))
        return redirect(rlunparse(urlparts), 301)
 ```
this might it, but I don't have time to test it right now

**Comment by AndrewVSutherland** [comment]
padicField has been added to app.py

**Comment by AndrewVSutherland** [comment]
I believe that this has now been addressed (via #4540).  But anyone who disagrees should reopen it.
